### PR TITLE
Added different color feature

### PR DESCRIPTION
**Pull request by himanshujaidka**
Thanks to @imsahil007 👍🏻 :)

### CODE REVIEW

File: assets/js/sketch.js [hunk 1]
```diff
@@ -29,19 +29,36 @@ function setup() {
   rectMode(CORNER);
 }
 
+defInk = "#264180";//Default ink
```
Comment (smaranjitghose) [inline assets/js/sketch.js:32]: Specify the color in the comments as well. We want it to be more human-readable as well!

File: assets/js/sketch.js [hunk 2]
```diff
@@ -29,19 +29,36 @@ function setup() {
...
+
+function chooseBlue(){
+  defInk = "#264180"
+}
+
+function chooseRed(){
```
Comment (smaranjitghose) [inline assets/js/sketch.js:38]: Add comments or some function description please, later on it would become a big chunk of code and difficult to debug

**Comment by smaranjitghose** [comment]
Nice work @himanshujaidka !

Can you please make the small changes I suggeted
Also, add a pink color just for fun!

File: assets/js/sketch.js (hunk 2 above)
Comment (himanshujaidka) [reply assets/js/sketch.js:38]: ok I'll be

**Comment by himanshujaidka** [comment]
ok I'll be make comments

**Pull request merged**